azure-kusto-data/package.json

```diff
@@ -25,6 +25,7 @@
   "license": "ISC",
   "dependencies": {
     "adal-node": "^0.1.28",
+    "eslint": "^6.0.1",
```

Comment:
move to `devDependencies` please. and thanks for that !

Reply:
Fixed.